### DISCUSSION TITLE
photoshop's autocorrect

### DISCUSSION THREAD

**Discussion by lelldorin** [ImageMagick/ImageMagick#5802] [Digital Image Processing]
what commands are needed if you want to emulate photoshop's autocorrect for image enhancement?
I know normalize but it's not close enough yet.

I would like to add such a function to my gui

https://software.besly.de/images/IMGui1.png
_Imagemagick GUI on Haiku_

**Comment by fmw42** [comment]
Post your input photoshop file (zip it first) and your IM version and platform.

Also I do not see any "auto correct" only  auto tone, auto color and auto contrast in Photoshop.  Perhaps you can point out where the "auto correct" is located.

**Comment by lelldorin** [comment]
Oh sorry, thats right here the infos:

Photoshop Elements 8.0 (German: Überarbeiten:Intelligente Auto-Korrektur Alt+Strg+M)

Imagemagick on Haiku: Version 6.9.11 or 7.0

**Reply by fmw42** [reply]
I need more information or a picture. I do not see that in Photoshop 2023 (do not have Photoshop elements)
